packages/checkout-ui-extensions-run/src/dev.ts

```diff
@@ -1,6 +1,5 @@
 import {URL} from 'url';
 
-import getPort from 'get-port';
```

Comment:
The Ruby CLI is always passing a port. I removed the callback to avoid having to either create an `async` `parseDevelopmentServerConfig` or have the port handling done outside in `dev.ts`.